### PR TITLE
Add library functionality (PubMed-lookup) possibly

### PR DESCRIPTION
**Pull request by Anwesh1**

### CODE REVIEW

**Comment by JohnGiorgi** [comment]
Some comments after a quick glance (will update this after I have some more time):

1. We probably don't want to commit the raw data files to the repo (e.g. `pubmed_batch_20000_to_29999_abstracts.txt`). If you wanted to show these to us, you can just copy/paste a minimal example in a comment and format it with markdown.
2. We should explore `pubmed-lookup` or comparable tools as discussed, and specifically, figure out how they compare to your current script in terms of time/memory etc. Once we know this we can make a decision on whether to stick with the current script or delegate the fetching of data to another tool/lib.
3. During our meeting we decided to go with a single tab-separated file right? So this script should be producing a file where each line looks like: `pmid\ttitle\tabstract` (notice the `\t` characters).

File: requests_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import requests
+import json
+
+#getting start and end dates to get data from
+start = input("Enter start date in (YYYY/MM/DD) format: ")
+end = input("Enter end date in (YYYY/MM/DD) format: ")
+#using the esearch eutil to grab the 'Webenv' and 'total record' values to then send to efetch
+search_url = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&mindate="+start+"&maxdate="+end+"&usehistory=y&retmode=json"
+search_r = requests.post(search_url)
```
Comment (jvwong) [inline requests_test.py:9]: Ask me for an EUTILS API key. We should figure out a way to include this as a URL parameter by declaring the API key as a command-line or environmental variable.

Comment (jvwong) [reply requests_test.py:9]: Why use the HTTP POST method rather than GET?

**Review by jvwong — CHANGES_REQUESTED** [review]
By the way I have a module for using EFETCH in the semantic-search repo to retrieve title, abstract etc, it might be worth a look: https://github.com/PathwayCommons/semantic-search/blob/master/semantic_search/ncbi.py

File: requests_test.py (hunk 1 above)
Comment (Anwesh1) [reply requests_test.py:9]: The first few lines I got from a stack overflow and when I ran it I got good data, so I didn't to mess with it too much. That's why I didn't change the HTTP POST.

**Pull request closed without merging**